go.work

```diff
@@ -1,4 +1,6 @@
-go 1.23
+go 1.23.8
+
+toolchain go1.23.9
```

Comment:
Should we need to add toolchain...

Will it not cause an issue in GHAction ?

Reply:
will remove it